ci/set_version.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+set -euo pipefail
+set +x
+
+# Get the directory of the currently executing script and its parent dir
+current_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )";
+parent_dir="${current_dir%/*}"
+
+# pom xml files
+pom_xml_location="${parent_dir}"
+sdk_xml_location="${parent_dir}/sdk"
+dbtool_xml_location="${parent_dir}/tools/dbtool"
+simpleapp_xml_location="${parent_dir}/examples/simpleapp"
+sctool_xml_location="${parent_dir}/tools/sctool"
+current_pom_version="$(xpath -q -e '/project/version/text()' "${pom_xml_location}"/pom.xml)"
```

Comment:
Check xpath is installed, if not echo installation instructions (also, some SDK devs use Windows/git-bash).

Reply:
Using `mvn` command instead of xpath, so less things to install